gems/quilt_rails/test/controllers/quilt/ui_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'rails/generators'
+require 'generators/quilt/rails_setup/rails_setup_generator'
+
+module Quilt
+  class UiControllerTest < ActionDispatch::IntegrationTest
+    def run(*)
+      Rails.env.stub(:test?, false) { super } # NOTE: Bypass DoNotIntegrationTestError
```

Comment:
This is to bypass the guard we have to discourage integration testing on the route. If we don't want to test this I can remove it.

Reply:
this seems fine, do you think we should have a built in way to bypass this?

Reply:
Yeah I can make it so we don't need to stub before merging.

Reply:
Better to actually boot the app in production/development rather than stubbing this.

Reply:
I may be able to slash the configuration commit if that works. Let me check.